Cargo.toml

```diff
@@ -309,7 +323,16 @@ path = "examples/events/main.rs"
 [[example]]
 name = "rendy"
 path = "examples/rendy/main.rs"
-required-features = ["animation", "gltf"]
+required-features = ["animation", "tiles_benchmarkgltf"]
```

Comment:
typo? I don't see tiles_benchmark as a feature, just as a separate example (and if so, separate from gltf or is it just one)

Reply:
Yep, just a typo leftover from when I was testing and benchmarking.